src/develop/masks/ellipse.c

```diff
@@ -819,6 +1196,7 @@ static void dt_ellipse_events_post_expose(cairo_t *cr, float zoom_scale, dt_mask
 
   float x, y;
 
+  // TODO: this can be replaced with a call to dt_ellipse_draw_shape()
```

Comment:
and why it is not currently done? maybe add some comment?

Reply:
because I'm lazy, but I don't want to write that down on the code...
Now seriously, I wanted to make sure it works first and then I just forget, I'll do it on the next commit.

Reply:
Ok, fine with me. I just wanted to be sure this was not some leftover or forgotten part.